docs/faq/index.rst

```diff
@@ -1,17 +1,285 @@
 
-.. _the Flocker Google group: https://groups.google.com/forum/#!forum/flocker-users
+I’m getting an OpenSSL error when I start Flocker services, what should I do?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Make sure when your create your certificates that you use a DNS or IP name for the control service certificate.
+This will make communicating with the control service from your agent nodes easier in the long run.
+
+If you create your control service certificate with the name ``my-control-service``, then your :file:`agent.yml` must also reference the control service as ``my-control-service``.
+This means that you must make that name DNS resolvable in order to avoid ssl issues.
+If you use a DNS name or IP, then the configuration is more natural.
+
+For more information on authentication, see :ref:`authentication-standalone-flocker`.
+   
+If you have further issues with SSL, please contact support@clusterhq.com.
+
+Using Flocker
+-------------
+
+How do I integrate Flocker with Docker Swarm?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Docker Swarm is Flocker native!
+This means you can use our Docker volume driver to easily integrate with Swarm.
+
+Here are some useful links for more information:
+
+* :ref:`Using Flocker with Docker, Swarm, Compose <docker-integration>`
+* :ref:`about-docker-integration`
+
+
+How do I integrate Flocker with Kubernetes?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Here are some useful links for more information:
+
+* :ref:`Using Flocker with Kubernetes <kubernetes-integration>`
+* `Demo: High Availability with Kubernetes and Flocker <https://clusterhq.com/2015/12/22/ha-demo-kubernetes-flocker/>`_
+* `Tutorial: Deploying a Replicated Redis Cluster on Kubernetes with Flocker <https://clusterhq.com/2016/02/11/kubernetes-redis-cluster/>`_
+
+How do I integrate Flocker with Mesos?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can integrate Flocker with Mesos using either the Mesos-Flocker Isolator, or with Marathon and the Flocker plugin for Docker.
+
+.. note:: Using Flocker with Mesos is one of our experimental projects, developed to less rigorous quality and testing standards than the mainline Flocker distribution. It is not built with production-readiness in mind.
+
+Here are some useful links for more information:
+
+* :ref:`Using Flocker with Mesos <mesos-integration>`
+* `Mesos-Flocker Isolator <http://flocker.mesosframeworks.com/>`_
+* `Demo: High Availability with Marathon and Flocker <https://clusterhq.com/2015/10/06/marathon-ha-demo/>`_
+* `Github: Using Mesos isolator for Flocker <https://github.com/ClusterHQ/mesos-module-flocker>`_
+
+Can I run Flocker with local storage?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Flocker integrates with many storage platforms including software defined storage platforms like EMC ScaleIO, Hedvig, Ceph, and ConvergIO.
```

Comment:
Good spot - cheers. Surprised the spell checker didn't pick this up
On 4 Apr 2016 23:43, "Marcus Ewert" <notifications@github.com> wrote:

> In docs/faq/index.rst
> <https://github.com/ClusterHQ/flocker/pull/2732#discussion_r58461604>:
>
> > +
> > +You can integrate Flocker with Mesos using either the Mesos-Flocker Isolator, or with Marathon and the Flocker plugin for Docker.
> > +
> > +.. note:: Using Flocker with Mesos is one of our experimental projects, developed to less rigorous quality and testing standards than the mainline Flocker distribution. It is not built with production-readiness in mind.
> > +
> > +Here are some useful links for more information:
> > +
> > +* :ref:`Using Flocker with Mesos <mesos-integration>`
> > +* `Mesos-Flocker Isolator <http://flocker.mesosframeworks.com/>`_
> > +* `Demo: High Availability with Marathon and Flocker <https://clusterhq.com/2015/10/06/marathon-ha-demo/>`_
> > +* `Github: Using Mesos isolator for Flocker <https://github.com/ClusterHQ/mesos-module-flocker>`_
> > +
> > +Can I run Flocker with local storage?
> > +^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> > +
> > +Flocker integrates with many storage platforms including software defined storage platforms like EMC ScaleIO, Hedvig, Ceph, and ConvergIO.
>
> Sorry to nit-pick, but I think it's actually ConvergeIO rather than
> ConvergIO. I'm just going to fix this.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ClusterHQ/flocker/pull/2732/files/b7a0c4af1ff8e4832b75486df7732b3fe8a3b981#r58461604>
>